<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidgetPlayer">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>771</width>
       <height>471</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>771</width>
       <height>471</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRemover">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Remover</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAtualizar">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>2</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Atualizar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <widget class="QMenu" name="menuJogadores">
    <property name="title">
     <string>Jogadores</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionNovo"/>
    <addaction name="actionAdicionar"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuJogadores"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionAdicionar">
   <property name="text">
    <string>Adicionar</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
